### DISCUSSION TITLE
Change multiple inline style attributes before typing

### DISCUSSION THREAD

**Discussion by jameslesliewatts** [ueberdosis/tiptap#2038] [General Questions] [answered]
First of all, thank you - v2 is awesome!

I'm having an issue setting multiple stored marks before typing. For example, if I change the font family and then change the color, color will change but font family will revert. It also happens if you change alignment after changing a style(although setting the alignment first and then changing a style is fine) and with a custom font size extension I made that is modeled on the color extension.

I made a simple sandbox [here](https://codesandbox.io/s/tiptap-issue-template-forked-jclnu?file=/src/components/Tiptap.vue) to test this.

Is there a way around the issue or something I'm doing incorrectly? Should I make custom extensions for each of these that will wrap the existing content in a new span with one inline style each?

**Comment by philippkuehn** [comment]
Oh, I think this is a bug.

**Reply by jameslesliewatts** [reply]
@philippkuehn Would it be helpful if I created an issue for this?

**Reply by philippkuehn** [reply]
Yes! I think I can fix this later today.

**Comment by philippkuehn** [comment, marked as answer]
fixed in the latest release.

**Reply by jameslesliewatts** [reply]
That's incredible! Thank you for being so responsive.

**Reply by mzmpy** [reply]
This issue still arises now. When my text is set to be bold, italic and underline, I want to unset bold and then italic following but finally only unset italic actually.
